.env.example

```diff
@@ -1,7 +1,6 @@
 # Network, possible values: development, pacific, rinkeby, mainnet
 GATSBY_NETWORK="rinkeby"
-
-#GATSBY_INFURA_PROJECT_ID="xxx"
+GATSBY_INFURA_PROJECT_ID="a983b53583044593956054de049922fd"
```

Comment:
please never ever commit credentials. This infura Id should be now considered exposed and compromised, so will invalidate it in infura

Reply:
Right, sorry about that